src/components/Creators/Testimonials.vue

```diff
@@ -156,16 +114,7 @@
                   </div>
                   <div class="ml-4 lg:ml-0">
                     <div
-                      class="
-                        text-xl
-                        font-medium
-                        text-gray-900
-                        dark:text-gray-100
-                        underline
-                        inline-flex
-                        items-center
-                        justify-around
-                      "
+                      class="text-xl font-medium text-gray-900 dark:text-gray-100 underline inline-flex items-center justify-around"
```

Comment:
For util classes you can just use the built in util classes. Plus the new updated v0.1.1 there's layout util classes now.